applications/DEM_application/custom_utilities/move_mesh_utility.h

```diff
@@ -0,0 +1,45 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Klauss B Sautter (based on Riccardo Rossi's work)
+//
+//
+
+#if !defined(KRATOS_MOVE_MESH_UTILITY_H_INCLUDED )
+#define  KRATOS_MOVE_MESH_UTILITY_H_INCLUDED
+
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "includes/define.h"
+#include "includes/model_part.h"
+
+
+namespace  Kratos
+{
+    class MoveMeshUtility
```

Comment:
What purpose does this class serve that a function cannot?

Reply:
there might be more functions coming but is it a problem to do it in this way?

Reply:
It's not necessarily a problem. It's more of a question to think critically about the design. At a minimum the class structure is adding complexity so it is worth to ask what is the reason.